app/views/schools/estimate_participants/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+Which heading do we want ?
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <!-- have we a helper for cohort year, yes -->
+    <h2 class="govuk-heading-l">Estimated numbers of participants starting in 2021</h2>
```

Comment:
You can use the cohort.start_year rather than hardcoding 2021

Reply:
Yes, I found it